book.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './person'
+class Book
+  attr_reader :borrowings
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @borrowings = []
+  end
```

Comment:
- [ ] According to the many-to-many relationship (A book has many rentals), a method for adding rentals should be present in the Book class. Check out the example in a previous lesson: [many-to-many relationship](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more information.